packages/xo-web/src/common/intl/messages.js

```diff
@@ -803,6 +803,13 @@ const messages = {
   srShared: 'Shared',
   srNotShared: 'Not shared',
   pbdNoSr: 'No storage detected',
+  deletePbdModalTitle: 'Delete PBD',
+  deletePbdModalMessage:
+    'Are you sure you want to forget this PBD? This will disconnect the SR from all their hosts by removing their links to them (PBDs)',
```

Comment:
Are you sure you want to forget this host?
This will disconnect the SR from the host by removing its link to it (PBD).